pom.xml

```diff
@@ -86,9 +86,9 @@
   </prerequisites>
 
   <scm>
-    <connection>scm:svn:http://svn.codehaus.org/mojo/trunk/mojo/mojo-parent</connection>
-    <developerConnection>scm:svn:https://svn.codehaus.org/mojo/trunk/mojo/mojo-parent</developerConnection>
-    <url>http://fisheye.codehaus.org/browse/mojo/trunk/mojo/mojo-parent</url>
+    <connection>scm:svn:https://github.com/mojohaus/mojo-parent</connection>
+    <developerConnection>scm:svn:https://github.com/mojohaus/mojo-parent</developerConnection>
```

Comment:
Thanks, but didn't you forget to also update the protocol part? (from svn to Git).
Cheers